P14-C04-training-with-multi-machines.ipynb

```diff
@@ -6,13 +6,13 @@
    "source": [
     "# Training with multi-machines\n",
     "\n",
-    "On previous two tutorials we saw that using multiple GPUs within a machine can accelerate the training. The speedup, however, is limited by the number of GPUs installed in that machine, which is typically at most 16 nowadays. This amount of speedup is still not good enough for large scale applications, such as training a state-of-the-art CNNs on millions of images.\n",
+    "On previous two tutorials we saw that using multiple GPUs within a machine can accelerate the training. The speedup, however, is limited by the number of GPUs installed in that machine, which is typically at most 16 nowadays. This amount of speedup is still not good enough for large scale applications, such as training a state-of-the-art CNN on millions of images.\n",
     "\n",
-    "This tutorial we discuss the key ideas how to turn a single machine training program into distributed training. A typical distributed system is illustrated in the following figure, where multiple machines are connected by network switches.\n",
+    "In this tutorial we discuss the key ideas on how to turn a single machine training program into distributed training. A typical distributed system is illustrated in the following figure, where multiple machines are connected by network switches.\n",
```

Comment:
Add a comma after the transitional phrase:
-> "In this tutorial, we discuss..."